modules/vantagecloud-lake/pages/vantagecloud-lake-demo-jupyter-google-cloud-vertex-ai.adoc

```diff
@@ -0,0 +1,152 @@
+= Run Teradata Jupyter Notebook Demos for VantageCloud Lake in Google Cloud Vertex AI  
+:experimental:
+:page-author: Janeth Graziani
+:page-email: Janeth.graziani@teradata.com
+:page-revdate: January 24, 2024
+:description: Run VantageCloud Lake Demos using Jupyter notebooks in Google Vertex AI Workbench. 
+:keywords: data warehouses, compute storage separation, teradata, vantage, cloud data platform, business intelligence, enterprise analytics, jupyter, teradatasql, ipython-sql, teradatasqlalchemy, vantagecloud, vantagecloud lake, data analytics, data science, vertex ai, google cloud
+:dir: vantage-lake-demo-jupyter-google-cloud-vertex-ai
+
+== Overview
+This quickstart explains how to run https://github.com/Teradata/lake-demos[Teradata Jupyter Notebook Demos for Vantage Cloud Lake] on Vertex AI, the AI/ML platform for Google Cloud. 
+
+== Prerequisites
+* Teradata modules for Jupyter **Linux desktop version** (download https://downloads.teradata.com/download/tools/vantage-modules-for-jupyter[here], registration required)
+* Google Cloud account with https://console.cloud.google.com/flows/enableapi?apiid=notebooks.googleapis.com,aiplatform.googleapis.com&redirect=https://console.cloud.google.com&_ga=2.180323111.284679914.1706204112-1996764819.1705688373[Vertex AI and Notebooks API] enabled
+* Google cloud storage to store startup scripts and Teradata Jupyter extension package
+* https://quickstarts.teradata.com/getting-started-with-vantagecloud-lake.html[Access to a VantageCloud Lake environment]
+
+== Vertex AI Google Cloud environment setup
+
+When we create a new notebook instance, we can specify a startup script. This script, which runs only once after instance creation, will install the Teradata Jupyter extension package and clone a GitHub repository into the new user-managed notebooks instance.
+
+* 1. Download Teradata Jupyter extensions package
```

Comment:
We should either use bullet point lists or numbered lists, but not both at the same time :). If it's a step by step procedure, it's best to use a numbered list.